src/main/io/displayport_msp_bf_compat.c

```diff
@@ -65,29 +65,30 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
 /*
     case SYM_HEADING:
         return BF_SYM_HEADING;
+*/
 
     case SYM_SCALE:
-        return BF_SYM_SCALE;
+        return 'I';
 
     case SYM_HDP_L:
-        return BF_SYM_HDP_L;
+        return 'L';
 
     case SYM_HDP_R:
-        return BF_SYM_HDP_R;
-*/
+        return 'R';
+
     case SYM_HOME:
         return BF_SYM_HOMEFLAG;
 
     case SYM_2RSS:
         return BF_SYM_RSSI;
 
-/*
     case SYM_DB:
-        return BF_SYM_DB
+        return 'D';
```

Comment:
The problem with using letters is that now you have two different osd items with D as symbol.

Reply:
I know But otherwise I habe? And i Do Not know any thing. So its maybe minimum some. Percent better then having only "?"

Reply:
Having L and R for HDOP makes no sense whatsoever.